0_hello_world/main_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := "Hallo du schöne Welt!\\n" // escape newline for easier string comparison
```

Comment:
the meaning of this comment escapes me. How is it easier to compare a string with an escaped newline than without?

Is it just so that the error output is more readable when the test fails? If so, `strconv.Quote()` might be a better way to do that in general.